### ISSUE TITLE
Navigation problems with a tablet

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#958]
A user has reported at youtube a navigation problem with our keymap when using his huion 1060. He has problems with orbiting using the pen.

We need to check for conflicts and differences to the Blender keymap. The tablet works fine with the Blender keymap. So it's our keymap that is in trouble.

The difficult part here is that i don't have a tablet :/

**Comment by ReinerBforartists** [comment]
Not to catch . The entries for the ndof device are all equal. I have no idea where the problem could reside. And as told, i cannot test it, i don't have a tablet.

**Comment by Draise14** [comment]
I do have a tablet. What would be the troubleshooting process?

**Comment by ReinerBforartists** [comment]
Ah, cool. Orbiting in the 3D view seems to be troubled. So the first step would be to find out which of the entries is responsible for orbiting. And then you need to find out why it doesn't work ^^

**Comment by Draise14** [comment]
Sure, I'll give it a go. What is the video the mentioned this?

**Comment by ReinerBforartists** [comment]
The problem is mentioned in the comments of this video here: https://www.youtube.com/watch?v=S-pyeCgTylA&lc=Ugy2xtA0iheMln_ZKJF4AaABAg

**Comment by Draise14** [comment]
I tried my tablet Wacom Bamboo CTH470. I have no problem orbiting (rotation of the viewport)

![2019-06-21_15-20-58](https://user-images.githubusercontent.com/35357409/59949307-56fcd380-9438-11e9-87ff-47b1e8293589.gif)

The way to do this is by pressing the Rclick on the pen while you drag and draw, this will orbit it. Not problem. I think he may have expected a modifier hotkey press to orbit, but it's just Rclick and drag - per normal. I found no problem.
